bootstrap.sh

```diff
@@ -1,10 +1,22 @@
 #!/bin/sh
 
+set -e
+
+export GOBIN=$(pwd)/$(go env GOHOSTOS)_$(go env GOHOSTARCH)/bin
```

Comment:
I don't think we should be setting globals that we haven't declared. this is going to confuse the hell out of people when their binaries start showing up under `harvey/linux_amd64/bin`, even after they stop playing with harvey. 

IMHO `HARVEYBIN` would be better and wouldn't have as many side effects.

Reply:
it's set *in this script*, for this script and its children. That's not global, right?

Reply:
GOBIN is for use by the go command. 

I'm a bit lost on the problem of setting a variable in a script that's inherited by its children. This is not a global. export is for a process and its children. I don't see the issue here.

Reply:
Do you mean that it might cause confusion at some point later on when extending the script @sevki? I.e., global from the script's perspective and all its child processes inheriting it?

Reply:
No, I screwed up. I was reviewing this on my phone and saw the entire change as a single block. I missed the folded lines.